modules/collector-source/pkg/util/interval.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/opencost/opencost/core/pkg/util/timeutil"
 )
 
-var intervalRegex = regexp.MustCompile(`^(\d+)(m|h|d|w)$`)
+var intervalRegex = regexp.MustCompile(`^(\d+)(s|m|h|d|w)$`)
```

Comment:
There is a function in our timeutil `ParseDuration` that is basically a one for one with go's `time.ParseDuration` except that it adds support for `w` and `d` I think. 

Honestly, this seems different enough that it doesn't really matter whether we reuse that package, but thought I'd at least mention it.